crates/symbolicator-service/src/services/symbolication/sourcemap.rs

```diff
@@ -172,9 +172,57 @@ fn fold_function_name(function_name: &str) -> String {
     format!("{folded}.{tail}")
 }
 
+/// Trims a line down to a goal of 140 characters, with a little wiggle room to be sensible
+/// and tries to trim around the given `column`. So it tries to extract 60 characters
+/// before and after the provided `column` and yield a better context.
```

Comment:
According to the code its actually 60 before, and 80 after the provided `column`.
Are we okay with all these values hardcoded?

Reply:
For now it's perfectly fine, especially that if we ever change it, it'd invalidate all our visual snapshot tests.